en/advanced_config/bootloader_update_from_betaflight.md

```diff
@@ -50,12 +82,25 @@ The button can be released after the board is powered up.
 
 ##### dfu-util
 
+To flash the bootloader onto the flight controller:
+
 ```
 dfu-util -a 0 --dfuse-address 0x08000000 -D  build/<target>/<target>.bin
 ```
 
 Reboot the flight controller and it let it boot without holding the boot button.
 
+:::note
+The [Holybro Kakute H7 v2](../flight_controller/kakuteh7v2.md) and mini flight controllers may require that you first run an additional command to erase flash parameters (in order to fix problems with parameter saving):
+
+```
+dfu-util -a 0 --dfuse-address:force:mass-erase:leave 0x08000000 -D  build/<target>/<target>.bin
+```
+
+The command may generate an error which can be ignored.
+Once completed, continue to flash the bootloader as previously mentioned.
```

Comment:
This reads as though for the problem FCs you must erase then flash. Is that right? Perhaps we should have this step first then?

Reply:
Correct, the erase operation must be completed first. I just didn't want to place the exception first, but placing it in a note seems to keep it out of the way.

Reply:
Yes, I see your point. I think this works and worse case people just do what is in the note, it does not harm.

Thanks again, merging now :-)